README.md

```diff
@@ -1,16 +1,52 @@
-![Cuprate](misc/logo/wordmark/CuprateWordmark.svg)
+<div align="center">
+	<img src="misc/logo/wordmark/CuprateWordmark.svg" width="50%"/>
 
+An alternative Monero node implementation.
```

Comment:
I suggest we decide on a tagline for Cuprate, and that our repo's "About" gets changed to it as well.

Not sure if `alternative` is a good choice for the long-term :)

Reply:
the immediate alternative is just `Rust` although it is a bit boring

Reply:
*Cuprate, an upcoming experimental, modern & secure monero node. Written in Rust * ?

Reply:
It's literally the project description and It's the tagline we used from the beginning

Reply:
I meant the tagline used for all time, i.e.:
- `Monero: the secure, private, untraceable cryptocurrency`.
- `Zcash - Financial Privacy in Rust 🦓`

> modern & secure

Not sure how far we should lean into this aspect (at least in the tagline), especially `secure` since that may imply that `monerod` is not secure.

> Written in Rust

I'm 50/50 on this, although, a lot of other projects include this, `arti`, `reth`, `zebra`.

Reply:
The *secure* originally came from the promised security features, isolation and traffic obfuscation, if this disturb you then what about: *An upcoming experimental, modern and performant Monero node*. no *secure*, we just stick to *modern* since we have a completely different architecture, I don't think its unrealistic to imply that monerod codebase is no more modern, tho it isn't the primary intent of the word.
> I'm 50/50 on this, although, a lot of other projects include this, arti, reth, zebra.

Honestly, I think first thing people wanna know about a rewrite is in which language it has been rewrite to + Rust is a selling point, so definitely have its place in the tagline imo

Reply:
> An upcoming experimental

Tagline for all time (present tense) so it would maybe be `A modern and performant Monero node`, although one day Cuprate will no longer be 'modern and performant', so I'd vote for a tagline that's more of a description that lasts forever like:

- `Arti - An implementation of Tor, in Rust.`
- `Zcash - Financial Privacy in Rust 🦓`

> Honestly, I think first thing people wanna know about a rewrite is in which language it has been rewrite to

I'm okay with/without so up to @Boog900.

Reply:
I don't think we actively _need_ to include Rust in the tagline, if it sounds good I don't really care either way.

---

This is when my suggested name change could've come in handy (REXmr), it would've come with a tagline `Rust Enhanced XMR` AND a T-Rex logo :D

Way too late now though but I mean we could still go with `Rust Enhanced XMR` as the tagline.